modules/backend/widgets/Lists.php

```diff
@@ -328,7 +328,10 @@ public function prepareModel()
                 if ($this->isColumnRelated($column)) {
                     $table = $this->model->makeRelation($column->relation)->getTable();
                     $columnName = isset($column->sqlSelect)
-                        ? DbDongle::raw($this->parseTableName($column->sqlSelect, $table))
+                        ? ( DbDongle::getDriver() == 'sqlsrv' 
+							? DbDongle::raw("" . $sqlSelect . " + ', '")
```

Comment:
Ah, thanks!

Verzonden vanuit Outlook Mobile<https://aka.ms/blhgte>



On Wed, Mar 23, 2016 at 7:33 AM -0700, "Pásztor Gábor" <notifications@github.com<mailto:notifications@github.com>> wrote:


In modules/backend/widgets/Lists.php<https://github.com/octobercms/october/pull/1871#discussion_r57168656>:

> @@ -328,7 +328,10 @@ public function prepareModel()
>                  if ($this->isColumnRelated($column)) {
>                      $table = $this->model->makeRelation($column->relation)->getTable();
>                      $columnName = isset($column->sqlSelect)
> -                        ? DbDongle::raw($this->parseTableName($column->sqlSelect, $table))
> +                        ? ( DbDongle::getDriver() == 'sqlsrv'
> +                                                     ? DbDongle::raw("" . $sqlSelect . " + ', '")


change $sqlSelect to $column->sqlSelect

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/octobercms/october/pull/1871/files/40234d6c312c8d5b386d5fa392fb8867db7875e7#r57168656>